### DISCUSSION TITLE
How can errors from JavaScript/Typescript bubble up to Rust to gracefully fail?

### DISCUSSION THREAD

**Discussion by vinayak-kukreja** [denoland/deno#22241] [Q&A]
Hey, I was trying to find documentation regarding error handling with JS/TS and Rust integration with the help of Deno. Unfortunately, I was not able to find any relevant documentation. Could someone please point me in the right direction?

**Comment by bartlomieju** [comment]
Could you explain a bit more what you mean? Are you interested how we JS exceptions are converted to Rust errors when the program fails?
